README.md

```diff
@@ -171,6 +171,9 @@ Configurable options, shown here with defaults: Please note the configuration op
     set :puma_tag, fetch(:application)
     set :puma_restart_command, 'bundle exec puma'
     set :puma_service_unit_name, "puma_#{fetch(:application)}_#{fetch(:stage)}"
+    set :puma_systemctl, :system # accepts :user
```

Comment:
Looks like you accidentally dropped `_user` from the variable name in that last commit.

Reply:
Good spot - I've amended the commit. 

I started refactoring as `:system` is not a user!